src/state/slices/common-selectors.ts

```diff
@@ -296,8 +296,10 @@ export const selectAssetsBySearchQuery = createCachedSelector(
 
     // Filters by low market-cap to avoid spew
     const filteredAssets = sortedAssets.filter(asset => {
+      // We don't have the market cap for TCY, but we want to show it
+      if (asset.assetId === tcyAssetId) return true
```

Comment:
Can we haz any `thorchainChainId` `mayachainChainId` assets here instead? MAYA, CACAO, and ROON could be similarly affected